web/src/composebox_typeahead.ts

```diff
@@ -415,7 +415,7 @@ export function tokenize_compose_str(s: string): string {
     // after the first character.
     let i = s.length;
 
-    let min_i = s.length - 25;
+    let min_i = s.length - 40;
```

Comment:
If the previous commit has a failing test without this change, they should be in the same commit. Each test should have tests passing.